Tools/autotest/arducopter.py

```diff
@@ -156,19 +156,18 @@ def takeoff(self,
         self.hover()
         self.progress("TAKEOFF COMPLETE")
 
-    def land_and_disarm(self, timeout=60):
+    def land_and_disarm(self, timeout=60, min_alt=6):
```

Comment:
What was the reason for this change? Why was this necessary for `Rangefinder()` autotest?

Reply:
This itself wasn't required (if I remember correctly) this was just to make the hard coded default min alt into a parameter that could be overridden. The change below at line 4384 was the one that was causing the test to fail.

Reply:
How does that work?  By passing 0 through for the minimum altitude you stay in the "wait for altitude" loop for longer, meaning you get an altitude countdown all the way to the ground.  But it shouldn't do anything except make the test *less* likely to pass as you now require an altitude of 0 before you'll start to check the arm status.

Only thing I can think of is a timeout was being triggered.

Note that if we *can* use 0 here it's all good by me, but I would like to know why the test was failing with it at the default of 6.

Reply:
This was due to the fall through that I dropped. Removed.